tools/topic_tools/src/throttle.cpp

```diff
@@ -58,6 +62,9 @@ ros::Time g_last_time;
 bool g_use_wallclock;
 bool g_lazy;
 ros::TransportHints g_th;
+ros::WallDuration g_wait_for_subscribers_timeout{ 1.0 };
+ros::WallDuration g_wait_for_subscribers_connect_time{ 1.0 };
+vector<string> g_wait_for_subscribers = { "*" };
```

Comment:
Why not use an empty list as the default?

Reply:
I was actually changing the previous behaviour under the assumption the new behaviour was better. That was the reason.

I've changed it. Actually all the defaults, so now the behaviour of this change should match the previous one.